spec/features/can_registred_spec.rb

```diff
@@ -25,4 +25,40 @@
     click_on 'Регистрация'
     expect(page).to have_content 'Email has already been taken'
   end
+
+  scenario 'the user has entered invalid email' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.